docs/conf.py

```diff
@@ -54,28 +54,44 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = 'sphinx_typlog_theme'
+html_theme = 'sphinx_book_theme'
 
-pygments_style = 'sphinx'
+pygments_style = 'algol_nu'
```

Comment:
Would love to get a slightly modified version of this with *just a bit* more contrast in the palette.